crypto/ocsp/ocsp_test.cc

```diff
@@ -644,6 +647,84 @@ TEST(OCSPTest, BasicAddCert) {
             cert.get());
 }
 
+TEST(OCSPTest, BasicAddStatus) {
```

Comment:
Any cherry-picked checks on the second mode of `OCSP_basic_add1_status()`?
> // 2. If |status| has the value of either |V_OCSP_CERTSTATUS_GOOD| or
// |V_OCSP_CERTSTATUS_UNKNOWN|, |revoked_reason| and |revoked_time| are ignored.

Reply:
It's ignored, so it'll still succeed so I hadn't thought of adding an additional check here. But will add.